sections/03_categorize.md

```diff
@@ -205,6 +234,12 @@ we see deep learning currently being used" section below.*
 expensive, so it may be helpful. I leave it here as a stub in case someone wants
 to take it on.*
 
+*(Wei Xie; @XieConnect): I personally do not think this subsection is necessary. 
```

Comment:
Even if it is not a problem, so we want to quickly state what @XieConnect describes regarding why it is not a problem in this domain?  I'm not sure where that would fit if it isn't a sub-section here.  It could be a brief remark in the 'Hardware limitations and scaling' sub-section of Discussion if storage is not a problem in any of the domains we cover.